pykg2tbl/j2/templates/skos-collection-member-count.sparql

```diff
@@ -0,0 +1,13 @@
+{# Jinja Template --> Sparql Query
+ | Count the number of members in a skos collection
+#}
+
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix dc:   <http://purl.org/dc/terms/> 
+SELECT ?coll ?ttl (count(?mem) as ?cnt) WHERE {
```

Comment:
as return please provide sensible names as they get used as column names in the end - so rather 
`SELECT (?coll as ?collection) (?ttl as ?title) (count(?mem) as ?count) WHERE`